src/SearchViewProvider.ts

```diff
@@ -152,27 +129,21 @@ export default class SearchViewProvider implements vscode.WebviewViewProvider {
                     <div class="loader js-loader">
                         <div class="loaderBar"></div>
                     </div>
-                    <div class="filters">
-                        <label>
-                            <input type="radio" name="filterBy" value="upcoming" checked class="js-filter" />
-                            current
-                        </label>
-                        <label>
-                            <input type="radio" name="filterBy" value="current" class="js-filter" />
-                            upcoming
-                        </label>
-                    </div>
                     <input 
                         class="js-query-input" 
-                        type="text" 
-                        title="Wildcards: query*   s*Model \nFields: title:OrderMgr \nFuzzy:  odrer~1  oderr~2 \nTerm presence: +getContent -pipelet -upcoming"
-                        placeholder="Search *query -pipelet +static fuzzy~3" />
+                        type="search" 
+                        placeholder="Search SFCC Docs" />
                 </div>
 
                 <div class="js-search-result-wrapper results"></div>
 
                 Additional resources:
                 <ul>
+                    <li>
```

Comment:
Added the SFCC Docs website to the list of resources.  Fine to skip this if you prefer, just seemed to make sense if you accepted this PR since the other panels would also be linking to that source website.